client/src/components/Navbar.js

```diff
@@ -19,7 +19,7 @@ function UserAuthNav(props) {
   if (props.auth && (!props.auth.googleID && !props.auth.facebookID)) {
     return (
       // if not logged in > show sign in options
-      <NavDropdown id="tools-dropdown" eventKey={5} title="Login">
+      <NavDropdown id="tools-dropdown" eventKey={5} title="LOGIN">
```

Comment:
Should we make the NavBar headings once signed in all caps as well, My Campaign and Sign Out, {user}?

Reply:
Yes, thanks! Changed those.